helm-charts/splunk-otel-collector/values.yaml

```diff
@@ -342,6 +347,22 @@ clusterReceiver:
   # existing fields can be disabled by setting them to null value.
   config: {}
 
+# Configurations for how Otel agents collect metrics from the k8s control plane. OTel agents collect metrics from the
+# Kubernetes control plane pods that reside on the same node. This integration works with high availability clusters
+# that have multiple master nodes.
+
+k8sControlPlane:
+  # If the {control_plane_component}.config fields below are left empty, then they will be populated by default values
+  # which are defined in templates/config/_otel-agent.tpl. The default config values will work with most out-of-the-box
+  # k8s clusters. If the default config values don't work for you (maybe because you use non-standard ports or custom
+  # SSL certificates), then you can populate the config field with a custom config to use instead of the default
+  # config. You can follow the links below for the receiver documentation for each control plane component to get all
+  # the supported config options.
+
+  apiserver:
+    # Receiver Documentation: https://docs.splunk.com/observability/gdi/kubernetes-apiserver/kubernetes-apiserver.html
+    config:
```

Comment:
Everything that is rendered in the configmap already can be overridden. So there is no need in adding this section, it's already configurable like this:
```
agent:
  config:
    receivers:
      smartagent/kubernetes-apiserver:
        config:
           ...
```

`autodetect.k8sControlPlane` is enough

Reply:
I see. I noticed we had many configurations overridable, but I wasn't if they were all overridable. Thanks.

Reply:
Removed this unneeded section.

Reply:
If we are keeping this approach for some time, please add a section in configuration docs about control plane with an example on how to reconfigure it

Reply:
Added a control plane section to the advanced-configuration.md.